File: src/test/java/aoc2017/Day19Test.java
import org.junit.Test;

import java.io.IOException;

import static org.junit.Assert.assertEquals;

public class Day19Test {
    @Test
    public void testPart1() {
        assertEquals("ABCDEF", Day19.part1(("     |          \n" +
                "     |  +--+    \n" +
                "     A  |  C    \n" +
                " F---|----E|--+ \n" +
                "     |  |  |  D \n" +
                "     +B-+  +--+ \n").split("\n")));
    }

    @Test
    public void testPart2() throws IOException, InterruptedException {
        assertEquals(38, Day19.part2(("     |          \n" +
                "     |  +--+    \n" +
                "     A  |  C    \n" +
                " F---|----E|--+ \n" +
                "     |  |  |  D \n" +
                "     +B-+  +--+ \n").split("\n")));

    }


}

File: src/test/java/aoc2018/Day19Test.kt
package aoc2018

import org.assertj.core.api.Assertions.assertThat
import org.junit.Test

class Day19Test {

    @Test
    fun part1() {
        val input = " /n ".split("\n")
        assertThat(Day19.part1(input)).isEqualTo(1)
    }

    @Test
    fun part2() {
        val input = " /n ".split("\n")
        assertThat(Day19.part2(input)).isEqualTo(2)
    }
}
